package.json

```diff
@@ -344,7 +343,7 @@
     "@metamask/phishing-controller": "^12.0.1",
     "@metamask/post-message-stream": "^8.0.0",
     "@metamask/ppom-validator": "0.34.0",
-    "@metamask/profile-sync-controller": "^0.2.1",
+    "@metamask/profile-sync-controller": "npm:@metamask-previews/profile-sync-controller@0.5.0-preview-c1e764d",
```

Comment:
I never know module aliasing was a thing!
Before I used to always find/replace all imports, this is much nicer ❤️